### PR TITLE
Fixing dead links for yaml example

### PR DESCRIPTION
**Pull request by arifC**
This PR fixes dead links in the document about conformance testing. The links now point to the new location of the e2e.yaml example.

### CODE REVIEW

**Comment by chuckha** [comment]
@arifC can you please ad a sign-off to this commit?

File: docs/conformance-testing.md [hunk 1]
```diff
@@ -39,5 +39,5 @@ To customize the set of tests that are run as part of the report, the following
 [4]: https://github.com/kubernetes/community/blob/master/contributors/devel/e2e-tests.md#conformance-tests
 [5]: plugins.md
 [6]: https://github.com/kubernetes/kubernetes/issues/49313
-[7]: ../plugins.d/e2e.tmpl
-[8]: ../examples/quickstart.yaml#L133
+[7]: ../examples/plugins.d/e2e.yaml
+[8]: ../examples/plugins.d/e2e.yaml
```
Comment (stevesloka) [inline docs/conformance-testing.md]: Could just delete this ref (#8) and use #7 since they are the same now.

**Comment by stevesloka** [comment]
Hey @arifC thanks for the PR. Could you fix the one comment and also sign the commit? Thanks!

**Comment by johnSchnake** [comment]
If this gets rebased onto master we can merge. Thanks for the fix.

**Comment by johnSchnake** [comment]
@arifC Sorry, it seems that the github UI has gotten in our way here. Can you squash/rebase manually so that you just have the one commit and it is signed off on? That way it can pass all the CI checks.

**Pull request closed without merging**